gamedata/tf2.utils.nosoop.txt

```diff
@@ -348,7 +348,7 @@
 				// "medic_bonesaw_hits"
 				"library"	"server"
 				"linux"		"@_ZN18CTFWeaponBaseMelee5SmackEv"
-				"windows"	"\x55\x8B\xEC\x83\xEC\x74\x57\x8B\xF9\xE8\x2A\x2A\x2A\x2A\x89\x45\xF8\x85\xC0"
+				"windows"	"\x2A\x2A\x2A\x2A\x2A\x2A\x57\x8B\xF9\xE8\x2A\x2A\x2A\x2A\x89\x45\xF8\x85\xC0"
```

Comment:
For what it's worth, I'm pretty sure this shouldn't be necessary these days &mdash; currently-supported SM versions all perform signature lookups on a copy of the original bytes.

Reply:
It was implemented in https://github.com/alliedmodders/sourcemod/pull/1642
It was mainly a concern back when SM 1.10 was still in use (up until it was no longer the main stable branch and an SDK update killed it).